dashboard/app/asset_storage.go

```diff
@@ -280,7 +280,12 @@ func (ad *buildAssetDeprecator) buildBugStatusPolicy(build *Build) (bool, error)
 			return true, nil
 		}
 	}
-	return false, nil
+	// If there are no crashes, but it's the latest build, they may still appear.
+	lastBuild, err := lastManagerBuild(ad.c, build.Namespace, build.Manager)
```

Comment:
This won't be executed many per deprecation handler, right? Otherwise I would can the latest builds in buildAssetDeprecator.

Also, potentially we can get crashes for last-but-one build (not latest). Ideally, we somehow ignore it as well.
But not sure how to do it here.
Perhaps we set Build.AssetsLastCheck a week in future in uploadBuild so that there is some grace period for fresh builds.

Reply:
In theory it's still possible that there's an instance that does not get rebuilt for quite a long time, so there are no new Builds. With the current code we at least do cover report deprecation.

If we ignore this theoretical possibility, then yes, just adding some grace period should be enough (we have some long broken kernels, but we at least submit new syzkaller commits and have a lot of crashes). Wdyt?

Reply:
The grace period can't be 100% reliable. But it's just the most practical solution I see. Do you see better options?

Reply:
This PR implements, I think, the most reliable solution, but it adds +1 query for each asset we analyze. I can modify it so that we query the latest build not for every asset, but for every build and just ignore the last one for the particular syz-manager. Then it will be just +1 query for each `Build` object we analyze.